### PR TITLE
added c_to_f, f_to_c, fizzbuzz, timer

### PR DESCRIPTION
**Pull request by KrasavaStep**

### CODE REVIEW

**Comment by UlasevichVadim** [comment]
Approve

**Comment by UlasevichVadim** [comment]
Approve

**Comment by AverburhSiarhei** [comment]
Approve

File: src/main/kotlin/Ermoshin_Stepan/TimerCommand.kt [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                println(x)
+            }
+        }
+    }
+
+    companion object {
+        const val TIMER_MILLIS : Long = 1000
+        const val MAX_TIMER_VALUE = 10
+        const val MIN_TIMER_VALUE = 5
+        const val ZERO = 0
```
Comment (vshat-tms) [inline src/main/kotlin/Ermoshin_Stepan/TimerCommand.kt]: ?

File: src/main/kotlin/Ermoshin_Stepan/TimerCommand.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                println("$x...")
+            }
+            else {
+                println(x)
+            }
+        }
+    }
+
+    companion object {
+        const val TIMER_MILLIS : Long = 1000
```
Comment (vshat-tms) [inline src/main/kotlin/Ermoshin_Stepan/TimerCommand.kt]: : Long

`1000l`

File: src/main/kotlin/Ermoshin_Stepan/TimerCommand.kt [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    override fun execute(text: String) {
+        var timerApex = ZERO
+        try {
+            timerApex = text.toInt()
+        } catch (ex: NumberFormatException){
+            println("$text isn't a number")
+        }
+
+        if (timerApex > MAX_TIMER_VALUE || timerApex < MIN_TIMER_VALUE) timerApex = MIN_TIMER_VALUE
```
Comment (vshat-tms) [inline src/main/kotlin/Ermoshin_Stepan/TimerCommand.kt]:
```
if() {

}```

File: src/main/kotlin/Ermoshin_Stepan/TimerCommand.kt (hunk 2 above)
Comment (vshat-tms) [inline src/main/kotlin/Ermoshin_Stepan/TimerCommand.kt]: timeout

**Pull request closed without merging**